rust/arrow/src/compute/kernels/mod.rs

```diff
@@ -30,3 +30,4 @@ pub mod sort;
 pub mod substring;
 pub mod take;
 pub mod temporal;
+pub mod unary;
```

Comment:
Should we call this something more general, like `arity` https://en.wikipedia.org/wiki/Arity

We'd probably want a binary function that takes 2 arguments, or maybe ternary if we're feeling cute enough to push it 😉

Reply:
Since this was only unary, I gave it that name. Note that we can lift all binary arithmetics into a generic like this one and place it there.

I do not have strong feelings here, but it is the first time I hear about `arity` xD

Reply:
No worries, whoever adds a ternary function can change the name.

Reply:
That would be backward incompatible, so it would be good to change it soon. I will change to arity. It is a fun name. Another option would be `generics`, but that is too generic. :)